examples/cnet-quic/quic-cli.c

```diff
@@ -114,7 +114,7 @@ static struct {
 struct {
     const char *path;
     int to_file;
-} *reqs;
+} * reqs;
```

Comment:
Remove the space between '* reqs' to get clang-format to pass. This should have been fixed with the per-commit hook running clang-format.